pyhive/tests/test_sqlalchemy_presto.py

```diff
@@ -48,7 +50,7 @@ def test_reflect_select(self, engine, connection):
             {"1": 2, "3": 4},  # Presto converts all keys to strings so that they're valid JSON
             [1, 2],  # struct is returned as a list of elements
             # '{0:1}',
-            '0.1',
+            Decimal('0.1'),
```

Comment:
Corrected this test case since it was already failing with SQLAlchemy 1.3 and 1.4.